mozdownload/treeherder.py

```diff
@@ -21,48 +24,77 @@
     'win64': {'build_os': 'win', 'build_architecture': 'x86_64'},
 }
```

Comment:
`PLATFORM_MAPS` seems to be only used in treeherder.py and I don't know if it makes sense to add a dependency on scraper to treeherder.py. Maybe move all the constants to their own module or to `__init__.py`? In any case, I think it's fine to keep it as is.